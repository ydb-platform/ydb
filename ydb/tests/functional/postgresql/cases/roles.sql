SELECT rolname FROM pg_roles;
