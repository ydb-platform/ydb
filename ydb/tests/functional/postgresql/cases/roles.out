SELECT rolname FROM pg_roles;
 rolname
-----
   postgres
   root
(2 rows)
